outputs/arbitrum.json

```diff
@@ -698,7 +697,6 @@
   "pools/50MAGIC-50wstETH-46cd": "0x46Cd168A23A1D8F50dD6Adced25400e52d107355",
   "pools/50MAI-50DAI-e4c3": "0xE4c3e9496ea894aaba9cA35B85A61d2a4F8a2709",
   "pools/50MOLANDAK-50USDT-fed1": "0xfeD111077E0905eF2B2fbf3060cFa9a34BaB4383",
-  "pools/50MORE-50GHO-a051": "0xa0511640a33Fa6a1204abf302a0386F4BB794AA6",
```

Comment:
why?

https://balancer.fi/pools/arbitrum/v2/0xa4760805a101a30dc1442291bc1b1f72aa3c8f4c0002000000000000000005c4

Reply:
was looking at the wrong $more pool

https://balancer.fi/pools/arbitrum/v2/0xa0511640a33fa6a1204abf302a0386f4bb794aa60002000000000000000005ba

<img width="360" alt="Screenshot 2024-11-19 at 20 56 40" src="https://github.com/user-attachments/assets/985e4f66-1d0b-4305-9b69-530bacbaac38">